packages/frontend/src/components/common/Filters/FilterRuleForm.tsx

```diff
@@ -129,9 +133,20 @@ const FilterRuleForm: FC<Props> = ({
 
             {isEditMode &&
                 (!onConvertToGroup ? (
-                    <ActionIcon onClick={onDelete}>
-                        <MantineIcon icon={IconX} size="sm" />
-                    </ActionIcon>
+                    <Tooltip
+                        label={isRequiredLabel}
+                        disabled={!isRequired}
```

Comment:
can you add `variant="xs"` - makes it smaller that's all. 
and perhaps a `multiline`

Reply:
done thanks